keps/sig-node/2837-pod-level-resource-spec/README.md

```diff
@@ -396,6 +400,37 @@ type PodSpec struct {
 }
 ```
 
+#### PodStatus API changes
+
+Extend `PodStatus` to include pod-level analog of the container status resource
+fields. Pod-level resource information in `PodStatus` is essential for pod-level [In-Place Pod
+Update]
+(https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/1287-in-place-update-pod-resources/README.md#api-changes)
+as it provides a way to track, report and use the actual resource allocation for the
+pod, both before and after a resize operation.
+
+```
+type PodStatus struct {
+...
+  // Resources represents the compute resource requests and limits that have been
+  // applied at the pod level. If pod-level resources are not explicitly specified,
+  // then these will be the aggregate resources computed from containers. If limits are
+  // not defined for all containers (and not defined at the pod level) then no aggregate
```

Comment:
is this saying
1. limits will not be *populated* in this field if they're not specified in spec.resources and we can't aggregate them from containers because not all containers specify limits
2. limits *will* be populated here but not applied/enforced at the pod level?

Reply:
Updated the language to clarity